### PR TITLE
fix(*): fixed HTML errors

### PR DESCRIPTION
**Pull request by Apollidore**

### CODE REVIEW

File: src/emailpro/emailpro-error-messages.popover.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <ul class="m-0">
     <li data-ng-repeat="message in messageDetails">
```
Comment (antleblanc) [inline src/emailpro/emailpro-error-messages.popover.html]: I think you could add `track by message.id`.

Comment (Apollidore) [reply src/emailpro/emailpro-error-messages.popover.html]: Good idea, except that `message.id` is sometimes missing, so I used `track by $index` instead

File: src/emailpro/account/alias/update/emailpro-account-alias-update.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
             </div>
 
             <div class="row pull-left mtop70" data-ng-show="noDomainMessage">
```
Comment (rfougier) [inline src/emailpro/account/alias/update/emailpro-account-alias-update.html]: Seems that update function isn't used anymore. Folder and file could be removed.

Comment (Apollidore) [reply src/emailpro/account/alias/update/emailpro-account-alias-update.html]: Fixed ! Deleted file !

**Pull request merged**